internal/cmd/cmd.go

```diff
@@ -31,7 +31,7 @@ var (
 
 	deDuplicateErrorRx = regexp.MustCompile(`:\s+`)
 	trailingSpaceRx    = regexp.MustCompile(` +\n`)
-	helpFlagsRx        = regexp.MustCompile("^### (?:`-([a-zA-Z])`, )?`--([a-zA-Z-]+)`")
+	helpFlagsRx        = regexp.MustCompile("^### (?:`-([0-9A-Za-z])`, )?`--([-0-9A-Za-z]+)`")
```

Comment:
Unless you plan on adding other numeric parameters, I’d just add `0` to the short version.

```suggestion
	helpFlagsRx        = regexp.MustCompile("^### (?:`-([0A-Za-z])`, )?`--([-A-Za-z]+)`")
```

Reply:
Thanks for the input. I want to keep the `0-9` in there as it's a bit more general than just the `0`. It took me quite a long time to work out that the reason I was getting a "flag not documented" panic on startup was because this regexp wasn't matching `-0`, and I don't want to repeat that debugging process :)